src/vmm/src/cpu_config/x86_64/cpuid/intel/normalize.rs

```diff
@@ -363,11 +370,11 @@ mod tests {
     )]
 
     use super::*;
-
     #[test]
     fn default_brand_string_test() {
+        const DEFAULT_BRAND_STRING_BASE: &[u8; 28] = b"Intel(R) Xeon(R) Processor @";
         let brand_string = b"Intel(R) Xeon(R) Platinum 8275CL CPU @ 3.00GHz\0\0";
-        let ok_result = default_brand_string(*brand_string);
+        let ok_result = default_brand_string(*brand_string, DEFAULT_BRAND_STRING_BASE);
         let expected = Ok(*b"Intel(R) Xeon(R) Processor @ 3.00GHz\0\0\0\0\0\0\0\0\0\0\0\0");
         assert_eq!(
             ok_result,
```

Comment:
I think it is worth changing these asserts, the simple `assert_eq!(ok_result, expected)` is enough.